.github/workflows/build-git-annex-macos.yaml

```diff
@@ -0,0 +1,216 @@
+        # <https://git-annex.branchable.com/bugs/gets_stuck_in_Remote_Tests___40____63____41___while_running_on_OSX_with_HOME_on_crippled_FS/?updated>
+        # is fixed
+      fail-fast: false
+    steps:
+      - name: Download git-annex package
+        uses: actions/download-artifact@v2-preview
+        with:
+          name: git-annex-macos-dmg-${{ needs.build-package.outputs.build-version }}
+
+      - name: Install git-annex package
+        run: |
+          hdiutil attach git-annex-*.dmg
+          rsync -a /Volumes/git-annex/git-annex.app /Applications/
+          hdiutil detach /Volumes/git-annex/
+          echo /Applications/git-annex.app/Contents/MacOS >> "$GITHUB_PATH"
+
+      - name: Print git-annex version
+        run: |
+          git annex version
+
+      - name: Checkout this repository for scripts/ci
+        uses: actions/checkout@v1
+
+      - name: Run tests
+        run: |
+          if echo "${{ matrix.flavor }}" | grep -q "crippled" ; then
+            # source so that CRIPPLEDFS_PATH will be set:
+            . scripts/ci/setup_crippledfs crippledfs 500
+          fi
+          case "${{ matrix.flavor }}" in
+            crippled-tmp)
+              # forces all test repos/paths into the VFAT FS
+              export TMPDIR="$CRIPPLEDFS_PATH"
+              ;;
+            crippled-home)
+              export HOME="$CRIPPLEDFS_PATH"
+              ;;
+            normal)
+              ;;
+            *)
+              echo "Unknown flavor ${{ matrix.flavor }}"
+              exit 1
+          esac
+
+          # Do it after we possibly setup HOME
+          git config --global user.email "test@github.land"
+          git config --global user.name "GitHub Almighty"
+
+          cd $HOME
+          export | grep -e crippledfs || :
+
+          git annex test
+
+  test-annex-more:
+    runs-on: macos-latest
+    needs: build-package
+    steps:
+      - name: Download git-annex package
+        uses: actions/download-artifact@v2-preview
+        with:
+          name: git-annex-macos-dmg-${{ needs.build-package.outputs.build-version }}
+
+      - name: Install git-annex package
+        run: |
+          hdiutil attach git-annex-*.dmg
+          rsync -a /Volumes/git-annex/git-annex.app /Applications/
+          hdiutil detach /Volumes/git-annex/
+          echo /Applications/git-annex.app/Contents/MacOS >> "$GITHUB_PATH"
+
+      - name: Seek of dynlibs
+        run: |
+          mkdir /tmp/testrepo; cd /tmp/testrepo; git init
+          function nfailed() {
+            strace -f git-annex "$1" 2>&1 | awk "/$2.*ENOENT/{print}" | tee /dev/fd/2 | wc -l
+          }
+          # We should get some reasonable number (not 40) of directories look up for dynamic libraries
+          liblookups=
+          PS4='> '; set -x
+          test $(nfailed version "libpcre.*so") -lt 5
+          test $(nfailed init "libpcre.*so") -lt 130
+
+  test-datalad:
+    # TODO: Re-enable macOS testing once it passes reliably
```

Comment:
hm, what was the issue? please add a URL here to reliability problem... 

also please space out in time (e.g. at 1h interval) cron jobs across workflows, so we do not start bombarding for all platforms at once

Reply:
During #33, we found that setting up Docker on the macOS environments would unpredictably fail with a message about GitHub rate limiting, even after following the steps in the Docker Machine readme for using a GitHub token.  When Docker didn't fail, the Datalad tests would end up with 9 errors and 2 failures.  As a result, [you said to disable testing of Datalad on macOS](https://github.com/datalad/datalad-extensions/pull/33#issuecomment-696368877).

Reply:
Link to comment added, and cron schedules adjusted.